deployment/terraform-ansible/aws/versions.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "3.3.0"
```

Comment:
Don't lock used version. It could be better if version is ">= 3.3.0"

Reply:
Version has been "unlocked" in new commit